elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/PlayerStats.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.example;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.datastores.aggregation.annotation.FriendlyName;
+import com.yahoo.elide.datastores.aggregation.annotation.MetricAggregation;
+import com.yahoo.elide.datastores.aggregation.metric.Max;
+
+import java.util.Date;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+/**
+ * A root level entity for testing AggregationDataStore.
+ */
+@Entity
+@Include(rootLevel = true)
+public class PlayerStats {
+
+    private String id;
+
+    private String overallRating;
+
+    private long highScore; // a metric
+
+    private Country country; // a dimension
+
+    private Date recordedDate;
+
+    @Id
+    public String getId() {
+        return id;
+    }
+
+    public void setId(final String id) {
+        this.id = id;
+    }
+
+    @FriendlyName
```

Comment:
FriendlyName wouldn't decorate the FactTable - it would decorate a field on a relationship from the Fact Table.   It would make sense to decorate country.name with this for example.